### ISSUE TITLE
Interface conversion error

### ISSUE THREAD

**Issue by holubec-petr** [txn2/kubefwd#114]
Hi,

starting from release 1.11.0 (valid also in current release), I'm getting this error sometimes:

```
panic: interface conversion: runtime.Object is *v1.Status, not *v1.Pod

goroutine 262 [running]:
github.com/txn2/kubefwd/pkg/fwdport.(*PortForwardOpts).WaitForPodRunning(0xc0000a0960, 0xc0004d0ae0, 0x0, 0x0, 0x0)
        /Users/cjimti/workspace/txn2/kubefwd/pkg/fwdport/fwdport.go:306 +0x61e
github.com/txn2/kubefwd/pkg/fwdport.(*PortForwardOpts).PortForward(0xc0000a0960, 0x0, 0x0)
        /Users/cjimti/workspace/txn2/kubefwd/pkg/fwdport/fwdport.go:130 +0x7d1
github.com/txn2/kubefwd/cmd/kubefwd/services.(*FwdServiceOpts).LoopPodToForward.func1(0xc0000a0960, 0xc0000a1680)
        /Users/cjimti/workspace/txn2/kubefwd/cmd/kubefwd/services/services.go:474 +0x39
created by github.com/txn2/kubefwd/cmd/kubefwd/services.(*FwdServiceOpts).LoopPodToForward
        /Users/cjimti/workspace/txn2/kubefwd/cmd/kubefwd/services/services.go:473 +0x923
```

First time it failed with this error on Evicted pod, but second time it failed on running pod without any problems.
Works as expected with version 1.10.0.

System: Windows 10 Pro 1803
kubectl client version: 1.18.0
kubectl server version: v1.14.9-eks
Command: kubefwd svc -x dev -n jenkins -n artifactory -n sonarqube

Thanks, Petr

**Comment by SorJEF** [comment]
Experiencing exactly the same issue with kubefwd 1.13.0

```
panic: interface conversion: runtime.Object is *v1.Status, not *v1.Pod

goroutine 86 [running]:
github.com/txn2/kubefwd/pkg/fwdport.(*PortForwardOpts).WaitForPodRunning(0xc0003fa3c0, 0xc00009ca80, 0x0, 0x0, 0x0)
        /Users/cjimti/workspace/txn2/kubefwd/pkg/fwdport/fwdport.go:306 +0x617
github.com/txn2/kubefwd/pkg/fwdport.(*PortForwardOpts).PortForward(0xc0003fa3c0, 0x0, 0x0)
        /Users/cjimti/workspace/txn2/kubefwd/pkg/fwdport/fwdport.go:130 +0x7ca
github.com/txn2/kubefwd/cmd/kubefwd/services.(*FwdServiceOpts).LoopPodToForward.func1(0xc0003fa3c0, 0xc0003fba40)
        /Users/cjimti/workspace/txn2/kubefwd/cmd/kubefwd/services/services.go:474 +0x32
created by github.com/txn2/kubefwd/cmd/kubefwd/services.(*FwdServiceOpts).LoopPodToForward
        /Users/cjimti/workspace/txn2/kubefwd/cmd/kubefwd/services/services.go:473 +0x91c
```

System: macOS Catalina 10.15.4
kubectl version: 1.18.0
server version: 1.15.9-gke.24

**Comment by calmkart** [comment]
fixed in #117

**Comment by cjimti** [comment]
built version 1.13.1
